wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -152,7 +156,8 @@ object Wallet extends CreateWalletApi with KeyHandlingLogger {
 
   def apply(mnemonicCode: MnemonicCode)(
       implicit config: WalletAppConfig,
-      ec: ExecutionContext): Wallet =
+      ec: ExecutionContext,
+      node: NodeApi): Wallet =
```

Comment:
I don't think this should be passed as an implicit every where as it is hard to follow implicits in my opinion.